connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -38,14 +38,17 @@ object KustoDataSourceUtils {
   val input: InputStream = getClass.getClassLoader.getResourceAsStream("app.properties")
   val prop = new Properties( )
   prop.load(input)
-  val version: String = prop.getProperty("application.version")
+  var version: String = prop.getProperty("application.version")
+  if(version == null){
+    version = prop.getProperty("version")
+  }
   val clientName = s"Kusto.Spark.Connector:$version"
-  val ingestPrefix: String = prop.getProperty("ingestPrefix")
-  val enginePrefix: String = prop.getProperty("enginePrefix")
-  val defaultDomainPostfix: String = prop.getProperty("defaultDomainPostfix")
-  val defaultClusterSuffix: String = prop.getProperty("defaultClusterSuffix")
-  val ariaClustersPrefix: String = prop.getProperty("ariaClustersPrefix")
-  val ariaClustersSuffix: String = prop.getProperty("ariaClustersSuffix")
+  val ingestPrefix: String = prop.getProperty("ingestPrefix","http://ingest-")
+  val enginePrefix: String = prop.getProperty("enginePrefix","http://")
+  val defaultDomainPostfix: String = prop.getProperty("defaultDomainPostfix","core.windows.net")
+  val defaultClusterSuffix: String = prop.getProperty("defaultClusterSuffix","kusto.windows.net")
+  val ariaClustersPrefix: String = prop.getProperty("ariaClustersPrefix", "https://kusto")
```

Comment:
where is this used?
Can't we just use the Aria proxy full URI: https://kusto.aria.microsoft.com ?

Reply:
ok lets